README.md

```diff
@@ -1,6 +1,7 @@
-# Deker local adapters
+# Deker Local Storage Adapters
```

Comment:
I'm sure, that there is no reason to rename the project, repo and package.

Reply:
+1

Reply:
Ok, rewritten to keep title